src/SoundList.js

```diff
@@ -32,13 +32,23 @@ const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
                   {track.username}
                 </span>
               </div>
-              <div className="hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0 space-x-12">
-                <IonIcon className="pr-4" icon={timeSharp} size="large" />
-                {timeConvert(track.duration)}
-                <IonIcon className="pr-4" icon={downloadSharp} size="large" />
-                {track.num_downloads}
-                <IonIcon className="pr-4" icon={playCircleSharp} size="large" />
-                {track.num_ratings}
+              <div className="w-6/12 hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0">
+                <div className="flex justify-start w-2/12 items-center">
+                  <IonIcon className="p-2" icon={timeSharp} size="large" />
+                  {timeConvert(track.duration)}
+                </div>
+                <div className="flex justify-start w-2/12 items-center">
```

Comment:
use `justify-end` instead of `justify-start`

Reply:
The only reason I used justify-start instead of justify end is because it made the first two columns very crowded looking. Let me go back and try again, thanks for the tip.

Reply:
> use `justify-end` instead of `justify-start`

Since I initially wasn't familiar with what either of these do, I read up on them. I'll share what I read in case it helps anyone reading this thread in the future.

[Tailwind's Justify Content documentation](https://tailwindcss.com/docs/justify-content) says that `justify-start` is shorthand for `justify-content: flex-start;` and `justify-end` is shorthand for `justify-content: flex-end;`.

Since [`justify-content: flex-start;` makes flex items "packed toward the start line" and `justify-content: flex-end;` makes items "packed toward to end line"](https://css-tricks.com/almanac/properties/j/justify-content/), using `justify-end` instead of `justify-start` would make flex items pack toward the end of the main axis instead of toward the start.

I found the below graphic from https://css-tricks.com/snippets/css/a-guide-to-flexbox/ really helpful.

![Screen Shot of graphic from https://css-tricks.com/snippets/css/a-guide-to-flexbox/ at 2020-11-18 at 8 59 16 PM](https://user-images.githubusercontent.com/43425812/99623418-ecf7ad80-29e0-11eb-9b85-0489f83c4b05.png)

Hope it helps!